frame/support/src/storage/types/counted_map.rs

```diff
@@ -133,8 +133,12 @@ where
 	}
 
 	/// Store or remove the value to be associated with `key` so that `get` returns the `query`.
```

Comment:
should we update the comment as fn no longer "*just* sets the key"?

Reply:
Sorry, haven't got what do you mean, can you elaborate ?
As I an see in the original implementation of the `StorageMap` we have the same description https://github.com/Mr-Leshiy/substrate/blob/2a5c7781c16b845e59c4d269b51f3f99228676e6/frame/support/src/storage/mod.rs#L174

Reply:
I meant that we should probably say "It decrements the counter when the value is removed"